components/Repositories/RossChart.tsx

```diff
@@ -60,6 +63,10 @@ export default function RossChart({ stats, rangedTotal, isLoading, error, range,
                 <h3 className="text-xs xl:text-sm text-slate-500">Average per day</h3>
                 <p className="font-semibold text-xl xl:text-3xl">{humanizeNumber(rangedAverage)}</p>
               </div>
+              <div>
+                <h3 className="text-xs xl:text-sm text-slate-500">Outside Contributors</h3>
```

Comment:
This copy isn't quite right: there could be _the same_ outside contributor contributing within all time buckets: i.e., if the same contributor contributed 4 times, that wouldn't be "4 outside contributors" that would be "4 outside contributions"

Reply:
Hmmmm maybe the ROSS index endpoint needs a "total" section since "recurring" _also_ includes contributors with the `CONTRIBUTOR` association. This:

```ts
return stats?.contributors.reduce((prev, curr) => (prev += curr.new), 0);
```

would only count the first time outside contributors who have not contributed before (i.e., those with association "NONE")